tests/XFTPCLI.hs

```diff
@@ -60,16 +71,24 @@ testXFTPCLISendReceive2servers = withXFTPServer . withXFTPServer2 $ do
   xftp ["rand", filePath, "19mb"] `shouldReturn` ["File created: " <> filePath]
   file <- LB.readFile filePath
   getFileSize filePath `shouldReturn` 19 * mb
-  let fd1 = filePath <> ".xftp" </> "rcv1.xftp"
-      fd2 = filePath <> ".xftp" </> "rcv2.xftp"
+  let fdRcv1 = filePath <> ".xftp" </> "rcv1.xftp"
+      fdRcv2 = filePath <> ".xftp" </> "rcv2.xftp"
+      fdSnd = filePath <> ".xftp" </> "snd.xftp"
```

Comment:
I don't like the idea of putting it in the same folder, it's a bit risky, also I was going to make that in case it's one recipient it will be a file, not a folder.

I suggest we name the file `filePath <> ".xftp.private"` or `filePath <> ".xftp.sender"` (with uniqueCombine obv.)